templates/cli/lib/commands/push.js.twig

```diff
@@ -1715,9 +1715,9 @@ push
     .action(actionRunner(pushMessagingTopic));
 
 const deploy = new Command("deploy")
-    .description(commandDescriptions['push'])
+    .description('Deprecated. Use appwrite push instead')
```

Comment:
Technically, if something is deprecated, it should still work, but it looks like we've removed the functionality.